docs/readmes/lte/dev_unit_testing.md

```diff
@@ -22,64 +22,39 @@ To SSH into the VM, run
 To run all existing unit tests, run
 
 ```bash
-[VM] cd magma/lte/gateway
-[VM] make test
+[VM] cd magma # or any subdirectory inside magma
+[VM] bazel test //...
```

Comment:
Is there a reason you kept the `[VM]`s here but got rid of them below?

Reply:
Just because of the `## Run all unit tests on the dev VM`, I suppose that would also apply to the ones below. Which do you prefer?

Reply:
I don't have a strong preference as long as it's consistent. That said, I don't think having `[VM]` hurts as a hint.

Reply:
Actually `In general, all unit testing for AGW is done on the magma dev VM.` is just really wrong. Maybe it's better if I just refactor this?